tests/unit/src/test/scala/tests/WorkspaceFoldersSuite.scala

```diff
@@ -32,6 +33,11 @@ class WorkspaceFoldersSuite
         expectError = false,
       )
       _ = assert(server.fullServer.folderServices.length == 2)
+      _ <- server.executeCommand(ServerCommands.ConnectBuildServer)
+      _ = assertNoDiff(
+        server.client.workspaceShowMessages,
```

Comment:
Wouldn't it make more sense to take assert on `workspaceMessageRequests` and see the that the correct choice popup appeared?